ykcompile/src/lib.rs

```diff
@@ -669,6 +677,14 @@ impl<TT> TraceCompiler<TT> {
     }
 
     fn c_binop(&mut self, dest: &IPlace, op: BinOp, opnd1: &IPlace, opnd2: &IPlace, checked: bool) {
+        let opnd1_ty = SIR.ty(&opnd1.ty());
```

Comment:
`opnd1_ty` is defined here.

Reply:
Ah, that's the next commit. I assume you'll squash later? Otherwise the first commit won't even compile.

Reply:
Oops. I didn't realise that. Yes, we can squash it all into one.